src/containers/AskGlific/AskGlific.module.css

```
@@ -0,0 +1,530 @@
+  padding: 1rem;
+  overflow-y: auto;
+  overflow-x: hidden;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.Message {
+  max-width: 80%;
+  font-size: 0.8rem;
+  word-wrap: break-word;
```

Stylelint (17.10.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)